src/alembic/versions/002_0d3bf1348c58_new_reference_org_name_columns.py

```diff
@@ -0,0 +1,36 @@
+"""new reference org name columns
+
+Revision ID: 0d3bf1348c58
+Revises: d630193252db
+Create Date: 2024-06-01 22:10:22.189820
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "0d3bf1348c58"
+down_revision: Union[str, None] = "d630193252db"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.add_column(
+        "use_transaction_longitudinal", sa.Column("reference", sa.String())
```

Comment:
question: is this going to break the import because this field doesn't exist?

Reply:
it doesn't. the `reference` field only shows up on May 22nd, but i was able to ingest data starting in May 1 without issue.

Reply:
The `schema_compare` function of `api_copy_job.py` will only throw an error is columns are found in the dmap api file that is being imported, but not found in the SQLAlchemy defined schemas.

Reply:
Could be worth throwing an explicit warning log for that...

Reply:
Added a task here for warning here: https://app.asana.com/0/1205813632253642/1207471258038093/f